packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -164,17 +162,17 @@ abstract class InteractiveInkFeatureFactory {
   /// [InteractiveInkFeature].
   @factory
   InteractiveInkFeature create({
-    @required MaterialInkController controller,
-    @required RenderBox referenceBox,
-    @required Offset position,
-    @required Color color,
-    @required TextDirection textDirection,
+    required MaterialInkController controller,
+    required RenderBox referenceBox,
+    required Offset position,
+    required Color color,
+    required TextDirection textDirection,
     bool containedInkWell = false,
-    RectCallback rectCallback,
-    BorderRadius borderRadius,
-    ShapeBorder customBorder,
-    double radius,
-    VoidCallback onRemoved,
+    RectCallback? rectCallback,
+    BorderRadius? borderRadius,
+    ShapeBorder? customBorder,
+    double? radius,
+    required VoidCallback? onRemoved,
```

Comment:
Tests in google are breaking because this was marked required.

@a14n do you remember why this needs to be required now?

/cc @Hixie @HansMuller

Reply:
Nothing obvious comes to my mind. We can remove the required keyword.

Reply:
Removing in https://github.com/flutter/flutter/pull/67427.